interface/modules/custom_modules/oe-module-faxsms/openemr.bootstrap.php

```diff
@@ -0,0 +1,194 @@
+<?php
+
+/**
+ * Bootstrap for custom Fax SMS module.
+ * Since this was our original example module,
+ * I've left it basically intact from original.
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ *
+ * @author    Stephen Nielson <stephen@nielson.org>
+ * @copyright Copyright (c) 2019 Stephen Nielson <stephen@nielson.org>
+ * @author    Jerry Padgett <sjpadgett@gmail.com>
+ * @copyright Copyright (c) 2019-2023 Jerry Padgett <sjpadgett@gmail.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+/*
+ This module uses an abstract class to arbitrate and dispatch
+ API calls to diffrent vendor services for both the fax and sms type on a per-call basis.
+ To add new vendors, just follow and use the existing dispatching flow
+ for an existing service type and vendor service.
+ */
+
+use OpenEMR\Events\Messaging\SendSmsEvent;
+use OpenEMR\Events\PatientDocuments\PatientDocumentEvent;
+use OpenEMR\Events\PatientReport\PatientReportEvent;
+use OpenEMR\Menu\MenuEvent;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\EventDispatcher\Event;
+
+// some flags
+$allowFax = ($GLOBALS['oefax_enable_fax'] ?? null);
+$allowSMS = ($GLOBALS['oefax_enable_sms'] ?? null);
+$allowSMSButtons = ($GLOBALS['oesms_send'] ?? null);
+
+/**
+ * @global OpenEMR\Core\ModulesClassLoader $classLoader
+ */
+$classLoader->registerNamespaceIfNotExists('OpenEMR\\Modules\\FaxSMS\\', __DIR__ . DIRECTORY_SEPARATOR . 'src');
```

Comment:
@sjpadgett or @adunsulag , just checking if this one line then allows use of your module classes in all scripts that include globals.php?
If so, then rec checking this out @claimrevolution for your claims module (can basically then drop all your composer stuff)

Reply:
Yep and it's pretty neat and time saving but like you say the main autoloader needs to be ran either via globals or directly.